### ISSUE TITLE
FreeBSD/PCBSD compile

### ISSUE THREAD

**Issue by Stuz719** [powertab/powertabeditor#226]
Instructions for compilation do not cover FreeBSD/PCBSD systems - not sure of any alterations for porting with regards to install directories or environment variables. Would be helpful to have instructions for any changes needed for clean installation and possible packaging as a pkg or pbi.

**Comment by cameronwhite** [comment]
I've never tried building on FreeBSD - I think the main issue would be seeing whether dependencies like rtmidi work.

**Comment by Stuz719** [comment]
AFAIK rtmidi should work

**Comment by ryan-sg** [comment]
I managed to get a git checkout as of a couple days ago to build on OpenBSD.  So, it should be fine on FreeBSD as well, I imagine.  I did have to manually pass various Qt5 cmake module directories to cmake to have it properly pick things up, and then I also had to make a minor edit to rtmidi to not try and pull in ALSA stuff (thus, no sound/midi support).  So it runs, it opens power tabs, and looks pretty damn good :-)

If still having issues, I can try and dig up the cmake command I used to get the cmake configure successful.  I would be interesting in working with devs to possibly add sndio support, but I think that just needs to be in rtmidi and nothing to do with the powertab editor itself, I think?  Any clarification appreciated.

(On that note, I noticed rtmidi says it can work via JACK, I know JACK is available on openbsd, perhaps I can shortcut my way to midi support via JACK?)
